ui/app/src/Views.elm

```diff
@@ -34,8 +34,9 @@ currentView model =
         AlertsRoute filter ->
             AlertList.view model.alertList filter
 
-        SilenceListRoute route ->
-            SilenceList.view model.silences model.silence model.currentTime model.filter
+        SilenceListRoute filter ->
+            -- Is this filter needed???
```

Comment:
I would like to keep it as filter since the actual value is a `Filter`. If you check out the alertlist and silencelist parsing, the url values get parsed into this record type and then that is what is being passed around.